firebase-appdistribution/test-app/test-app.gradle

```diff
@@ -25,8 +25,8 @@ android {
         applicationId "com.googletest.firebase.appdistribution.testapp"
         minSdkVersion 23
         targetSdkVersion 33
-        versionName "1.0"
-        versionCode 1
+        versionName "2.0"
+        versionCode 2
```

Comment:
Just checking: do you want to keep this in?

Reply:
Yes, I think we should get into the habit of increasing those numbers.
Adding the feedback functionality seems to enough for a major version bump.